src/QAT/qat/purr/backends/verification.py

```diff
@@ -89,25 +101,77 @@ def get_verification_model(qpu_type: QPUVersion) -> Optional[VerificationModel]:
         )
 
     if qpu_type.make == Lucy.__name__:
-        # TODO: Should have an apply_setup_to_hardware in live which people can use to build our own architecture.
-        model = VerificationModel(qpu_type, LucyVerificationEngine)
-        raise NotImplementedError("No lucy-specific model created yet.")
+        return apply_setup_to_hardware(
+            hw= VerificationModel(qpu_type, LucyVerificationEngine),
+            qubit_count=8,
+            pulse_hw_x_pi_2_width=25e-9,
+            pulse_hw_zx_pi_4_width=3.18e-07,
+            pulse_measure_width=2.41e-06,
```

Comment:
This just becomes:

`return apply_lucy_hardware(VerificationModel(qpu_type, LucyVerificationEngine))`